chrome/browser/resources/record_replay/record_replay.gni

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 The Chromium Authors
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# List of files that don't need to be passed to html_to_wrapper().
+non_web_component_files =
+    [
+      "record_replay.ts",
```

Comment:
this is where we'd add additional `.ts` files to include them in the `js` resource bundle.